src/tink/core/Error.hx

```diff
@@ -42,11 +43,15 @@ class TypedError<T> {
 	public var code(default, null):ErrorCode;
 	public var data(default, null):T;
 	public var pos(default, null):Null<Pos>;
+	public var callStack(default, null):Array<StackItem>;
+	public var exceptionStack(default, null):Array<StackItem>;
 	
 	public function new(?code:ErrorCode = InternalError, message, ?pos) {
 		this.code = code;
 		this.message = message;
 		this.pos = pos;
+		this.callStack = try CallStack.callStack() catch(e:Dynamic) [];
+		this.exceptionStack = try CallStack.exceptionStack() catch(e:Dynamic) [];

```

Comment:
Yep. I'd also like to suggest not make population of these fields configurable through some compiler flag. Particularly if an error is expected and there's already code in place to deal with it, adding this much payload may be wasteful.